bundles/specmate-dbprovider-h2/src/specmate/dbprovider/h2/H2Provider.java

```diff
@@ -42,16 +40,22 @@
 	private Pattern databaseNotFoundPattern = Pattern.compile(".*Database \\\".*\\\" not found.*", Pattern.DOTALL);
 
 	@Activate
-	public void activate() throws SpecmateException {
+	public void activate(Map<String, Object> properties) throws SpecmateException {
```

Comment:
With this change, the injection of the configurationAdmin service is not necessary anymore. Please remove the injection and instance variable.

Reply:
done